qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -198,11 +212,30 @@ def convert_drive(self, shift, instruction):
         """
         command_dict = {
             'name': instruction.command.name,
-            't0': shift+instruction.start_time,
+            't0': shift + instruction.start_time,
             'ch': instruction.channels[0].name
         }
         return self._qobj_model(**command_dict)
 
+    @bind_instruction(commands.ParametricInstruction)
+    def convert_parametric(self, shift, instruction):
+        """Return the converted `ParametricInstruction`.
+
+        Args:
+            shift (int): Offset time.
+            instruction (ParametricInstruction): An instance of a ParametricInstruction subclass.
+        Returns:
+            dict: Dictionary of required parameters.
+        """
+        command_dict = {
+            'name': 'parametric_pulse',
+            'pulse_shape': ParametricPulseShapes(type(instruction.command)).name,
+            't0': shift + instruction.start_time,
+            'ch': instruction.channels[0].name,
+            'params': instruction.command.get_params()
```

Comment:
I still think these should be parameterized by their index in a list rather than name. Name is very verbose and either makes room for unwanted complexity in parameters or is redundant. I guess this is a lost cause now though 😄.

Reply:
It doesn't make sense to use json and then try to make it efficient rather than readable. I'm all for transitioning to an ABI instead, but while we're using json, let's take advantage of its only benefit

Reply:
fair enough, I would love to implement a serialized binary format.